pkg/cmd/cli/upgradebackups/upgrade_backups.go

```diff
@@ -0,0 +1,263 @@
+package upgradebackups
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"log"
+	"path"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "github.com/heptio/velero/pkg/apis/velero/v1"
+	"github.com/heptio/velero/pkg/client"
+	"github.com/heptio/velero/pkg/cloudprovider"
+	"github.com/heptio/velero/pkg/persistence"
+	"github.com/heptio/velero/pkg/plugin"
+	"github.com/heptio/velero/pkg/util/encode"
+	"github.com/heptio/velero/pkg/volume"
+)
+
+/*
+
+// upgrade backups in the "default" backup location, and any snapshots
+// found will be written as being in the "aws-default" snapshot location
+velero upgrade --backup-location default --volume-snapshot-location aws-default
```

Comment:
Should this go into the `Example` field on the command?

Reply:
Oh, yeah, I think this was just a note for myself before I started coding. I think `Example` already captures this. Will remove.